lib/create-plugin.js

```diff
@@ -420,6 +421,16 @@ function recordResolveSegment(segment, scope) {
     const fieldNameMetric = `${RESOLVE_PREFIX}/${fieldName}`
     createMetricPairs(transaction, fieldNameMetric, scope, duration, exclusive)
   }
+
+  const fieldType = attributes[PARENT_TYPE_ATTR]
```

Comment:
@jordigh you should probably update the docs within the repo [here](https://github.com/newrelic/newrelic-node-apollo-server-plugin/blob/main/docs/metrics.md) to call out these new metrics